qdrant-landing/content/documentation/frameworks/genkit.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Firebase Genkit
+weight: 3400
+---
+
+# Firebase Genkit
+
+[Genkit](https://firebase.google.com/products/genkit) is a framework to build, deploy, and monitor production-ready AI-powered apps.
+
+You can build apps that generate custom content, use semantic search, handle unstructured inputs, answer questions with your business data, autonomously make decisions, orchestrate tool calls, and more.
+
+You can use Qdrant for indexing/semantic retrieval of data in your Genkit applications via the [Qdrant-Genkit plugin](https://github.com/qdrant/qdrant-genkit).
+
+Genkit currently supports server-side development in JavaScript/TypeScript (Node.js) with Go support in active development.
+
+## Installation
+
+```bash
+npm i genkitx-qdrant
+```
+
+## Configuration
+
+To use this plugin, specify it when you call `configureGenkit()`:
+
+```js
+import { qdrant } from 'genkitx-qdrant';
+
+export default configureGenkit({
+  plugins: [
+    qdrant([
+      {
+        clientParams: {
+          host: 'localhost',
+          port: 6333,
+        },
+        collectionName: 'some-collection',
+        embedder: textEmbeddingGecko,
```

Comment:
It would be great to mention where the `textEmbeddingGecko` variable comes from. How do we create it?

Reply:
[`cff9ecb` (#958)](https://github.com/qdrant/landing_page/pull/958/commits/cff9ecbdeec0776c61b1dfe75baf2497a8f51e6b)